mono/utils/mono-error.h

```diff
@@ -57,7 +57,7 @@ typedef union _MonoError {
 	struct {
 		uint16_t error_code;
 		uint16_t private_flags; /*DON'T TOUCH */
-		void *hidden_1 [12]; /*DON'T TOUCH */
+		void *hidden_1 [9]; /*DON'T TOUCH */
```

Comment:
This is a breaking change, since the struct is public, its size must remain constant.

Reply:
Oh because it in public embedding API? Agreed, will restore.